build_scripts/windows/scripts/build.cmd

```diff
@@ -122,7 +130,6 @@ set PYTHON_EXE=%PYTHON_DIR%\python.exe
 robocopy %PYTHON_DIR% %BUILDING_DIR% /s /NFL /NDL
 
 set CLI_SRC=%REPO_ROOT%\src
-%BUILDING_DIR%\python.exe -m pip install --no-warn-script-location --force-reinstall pycparser==2.18
```

Comment:
`pycparser==2.19` is installed later.

Reply:
Where?

Reply:
https://github.com/Azure/azure-cli/blob/6fa7aa45084d251a5b66b4d3c4dad1883b94d902/build_scripts/windows/scripts/build.cmd#L138

Reply:
Also see https://github.com/Azure/azure-cli/pull/27196#discussion_r1303879323